packages/office-ui-fabric-react/src/components/Fabric/Fabric.base.tsx

```diff
@@ -20,16 +21,13 @@ export class FabricBase extends React.Component<
   }
 
   public render() {
-    const { className } = this.props;
+    const classNames = this._getClassNamesHelper();
 
-    const classNames = getClassNames(getStyles, {
-      theme: this.props.theme!,
-      className,
-      isFocusVisible: this.state.isFocusVisible
-    });
     const divProps = getNativeProps<React.HTMLAttributes<HTMLDivElement>>(this.props, divProperties);
 
-    return <div {...divProps} className={classNames.root} ref={this._rootElement} />;
+    let fabricComponent;
+    fabricComponent = <div {...divProps} className={classNames.root} ref={this._rootElement} />;
+    return fabricComponent;
```

Comment:
nit: can be collapsed into `return <div...`

Reply:
done.